src/pcrlock/pcrlock.c

```diff
@@ -4955,6 +4962,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "policy",          required_argument, NULL, ARG_POLICY          },
                 { "force",           no_argument,       NULL, ARG_FORCE           },
                 { "entry-token",     required_argument, NULL, ARG_ENTRY_TOKEN     },
+                { "strict",          no_argument,       NULL, ARG_STRICT           },
```

Comment:
hmm, so i kinda sense that sooner or later we might want to have different modes of strictness. Hence I think it would be nicer to make this a `--strict=yes|no` argument, i.e. one that takes a parameter. Because that means we can later accept more than just yes/no.

use parse_boolean_argument() to implement this, it makes it trivial.

and thus switch this line from no_argument to required_argument. 

(in case you wonder optional_argument is not permitted in our codebase because ambigious what "--foo bar" means: could be that bar is value to `--foo`, or could be that it is the first positional argument).

also, } is not properly indented